### ISSUE TITLE
pertainym

### ISSUE THREAD

**Issue by arademaker** [globalwordnet/schemas#51]
In https://globalwordnet.github.io/schemas/ we have the 

The set of relations between senses is limited to the following

1. pertainym: A relational adjective. Adjectives that are pertainyms are usually defined by such phrases as “of or pertaining to” and do not have antonyms. A pertainym can point to a noun or another pertainym

Non-Princeton WordNet Relations:

2. pertainym: usually an adjective, which can be defined as “of or pertaining to” another word.

In the DTDs:

1. https://github.com/globalwordnet/schemas/blob/master/WN-LMF-relaxed-1.0.dtd#L195
2. https://github.com/globalwordnet/schemas/blob/master/WN-LMF-relaxed-1.1.dtd#L206

From https://globalwordnet.github.io/gwadoc/#pertainym, I got that this is the `\` in the PWN https://wordnet.princeton.edu/documentation/wninput5wn.

So in conclusion,  the page https://globalwordnet.github.io/schemas/ listed twice the same relation as two different relations but they are the same one, right?

**Comment by jmccrae** [comment]
It is both a sense and a synset relation. The synset usage of pertainym is beyond the usage in Princeton WordNet.

**Comment by arademaker** [comment]
I believe this issue should be closed once we improve the documentation... Yes, I know that Princeton used it as sense relation. But using the same name is confusing. What is the expected interpretation? DTDs declare `pertainym`  as sense relation.  But feel free to close again if you feel there is nothing to do here.

**Comment by arademaker** [comment]
Let me put it in other words. The `Non-Princeton WordNet Relations` can be used in an XML? How? The DTDs do not list the `pertainym` as one possible relation between synsets.

**Comment by goodmami** [comment]
I don't have an opinion on whether using the same name is confusing, but for documentation I wonder if [gwadoc](https://globalwordnet.github.io/gwadoc/) is a better place for such historical notes?

> The DTDs do not list the `pertainym` as one possible relation between synsets.

That's true. The Wn library does not allow `pertainym` between synsets because it's not in the DTD. Should it be there?

**Comment by jmccrae** [comment]
Right, the documentation is incorrect `pertainym` isn't a synset relation
